build.gradle

```diff
@@ -9,79 +9,200 @@
  * GitHub history for details.
```

Comment:
Is this needed?

Reply:
This is the standard header that @DarshitChanpura applied in the codebase in https://github.com/opensearch-project/security/pull/1829.  If you'd like to change the standard license, could we have that discussion in a sperate issue?